### PR TITLE
SLES - Updates

### PR DESCRIPTION
**Pull request by kiritigowda**
* Use AMD Clang ++ as default compiler
* Setup updates
* Readme updates
* changelog updates

### CODE REVIEW

File: samples/videoDecodeMem/CMakeLists.txt [hunk 1]
```diff
@@ -21,7 +21,7 @@
 #
 ################################################################################
 
-cmake_minimum_required (VERSION 3.5)
+cmake_minimum_required(VERSION 3.10)
```
Comment (rrawther) [inline samples/videoDecodeMem/CMakeLists.txt:24]: why is this changed to 3.10?

**Review by rrawther — CHANGES_REQUESTED** [review]
Added a comment

File: CMakeLists.txt [hunk 2]
```diff
@@ -21,13 +21,24 @@
...
+elseif(ROCM_PATH)
+  message("-- INFO:ROCM_PATH Set -- ${ROCM_PATH}")
+else()
+  set(ROCM_PATH /opt/rocm CACHE PATH "Default ROCm installation path")
+endif()
+
+# Set AMD Clang as default compiler
 if (NOT DEFINED CMAKE_CXX_COMPILER)
-    set(CMAKE_CXX_COMPILER clang++)
+  set(CMAKE_CXX_COMPILER ${ROCM_PATH}/bin/amdclang++)
```
Comment (AryanSalmanpour) [inline CMakeLists.txt:37]: please change `${ROCM_PATH}/bin/amdclang++`  to `${ROCM_PATH}/lib/llvm/bin/clang++`

**Review by AryanSalmanpour — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,35 @@
 
 Full documentation for rocDecode is available at [https://rocm.docs.amd.com/projects/rocDecode/en/latest/](https://rocm.docs.amd.com/projects/rocDecode/en/latest/)
 
+## rocDecode 0.9.0 (unreleased)
```
Comment (spolifroni-amd) [inline CHANGELOG.md]:
```suggestion
## (Unreleased) rocDecode 0.9.0
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,35 @@
...
+
+### Changed
+
+* AMD Clang++ is now the default CXX compiler.
+* `rocDecode-setup.py` setup script updates to common package install
```
Comment (spolifroni-amd) [inline CHANGELOG.md]: This needs more details. How did it behave before; how does it behave now; how does this affect the end user.

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,35 @@
...
+
+### Removed
+
+* 
+
+### Resolved issues
+
+* 
+
+### Tested configurations
```
Comment (spolifroni-amd) [inline CHANGELOG.md:20]: Tested configurations need to be removed from the changelogs. If the information is important for the end user, then let me know and I'll put it in the appropriate place in the documentation.

File: README.md [hunk 6]
```diff
@@ -29,7 +29,7 @@ access the video decoding features available on your GPU.
 > [!IMPORTANT]
 > `sudo amdgpu-install --usecase=rocm`
 
-* [Video Acceleration API](https://en.wikipedia.org/wiki/Video_Acceleration_API) - `Libva` is an implementation for VA-API
+* [Video Acceleration API](https://en.wikipedia.org/wiki/Video_Acceleration_API) - `libva-amdgpu-dev` is an AMD implementation for VA-API
```
Comment (spolifroni-amd) [inline README.md:32]: The prerequisites.rst doesn't have libva-amdgpu-dev mentioned specifically. Is that something that needs to be added?

**Review by spolifroni-amd — CHANGES_REQUESTED** [review]
There are a couple of minor changes needed in the changelog, and I have a question about a change in the readme that I didn't see in the prerequisites.rst file.

File: samples/videoDecodeMem/CMakeLists.txt (hunk 1 above)
Comment (kiritigowda) [reply samples/videoDecodeMem/CMakeLists.txt:24]: Version 3.5 will be deprecated and throw deprecation messages going forward. Version 3.10 is supported across all OS.

**Review by AryanSalmanpour — APPROVED** [review]

**Review by rrawther — APPROVED** [review]

**Comment by kiritigowda** [comment]
SLES

```
simoncat:~/test # ctest
Test project /root/test
    Start 1: video_decode-H265
1/7 Test #1: video_decode-H265 ................   Passed    5.86 sec
    Start 2: video_decodeMem-H265
2/7 Test #2: video_decodeMem-H265 .............   Passed    5.74 sec
    Start 3: video_decodePerf-H265
3/7 Test #3: video_decodePerf-H265 ............   Passed    6.49 sec
    Start 4: video_decodeRGB-H265
4/7 Test #4: video_decodeRGB-H265 .............   Passed   74.83 sec
    Start 5: video_decode-H264
5/7 Test #5: video_decode-H264 ................   Passed    5.73 sec
    Start 6: video_decodeBatch
6/7 Test #6: video_decodeBatch ................   Passed    8.10 sec
    Start 7: video_decode-AV1
7/7 Test #7: video_decode-AV1 .................   Passed    5.78 sec

100% tests passed, 0 tests failed out of 7

Total Test time (real) = 112.54 sec
simoncat:~/test # cat /etc/os-release 
NAME="SLES"
VERSION="15-SP5"
VERSION_ID="15.5"
PRETTY_NAME="SUSE Linux Enterprise Server 15 SP5"
ID="sles"
ID_LIKE="suse"
ANSI_COLOR="0;32"
CPE_NAME="cpe:/o:suse:sles:15:sp5"
DOCUMENTATION_URL="https://documentation.suse.com/"
```

File: CHANGELOG.md (hunk 4 above)
Comment (kiritigowda) [reply CHANGELOG.md]: updated

File: CHANGELOG.md (hunk 5 above)
Comment (kiritigowda) [reply CHANGELOG.md:20]: This would be a good to have detail for end-user

File: README.md (hunk 6 above)
Comment (kiritigowda) [reply README.md:32]: Updated

**Comment by kiritigowda** [comment]
@AryanSalmanpour - all updates done. Tests passing (logs attached). Upgraded the min rocm version required to 6.3 with deps updates.

File: CHANGELOG.md (hunk 5 above)
Comment (spolifroni-amd) [reply CHANGELOG.md:20]: OK. It'll be in the doc. We can remove it from here later.

**Review by spolifroni-amd — APPROVED** [review]

**Comment by kiritigowda** [comment]
@AryanSalmanpour  - ready to be merged.
